lane.py

```diff
@@ -1047,6 +1047,14 @@ def search(self, _db, query, search_client, pagination=None):
         # Get the search results from Elasticsearch.
         results = None
 
+        if not media:
+            media = self.media
+        else:
+            media = [media]
```

Comment:
It's not an issue now, but it's better to check whether this is already a list before turning it into a list. Someone could pass a list into `search`.

```
if isinstance(media, basestring):
    media = [media]
```